x-pack/plugins/apm/server/routes/settings/agent_configuration/route.ts

```diff
@@ -214,7 +214,7 @@ const createOrUpdateAgentConfigurationRoute = createApmServerRoute({
 
     if (resources.plugins.fleet) {
       await syncAgentConfigsToApmPackagePolicies({
-        core,
+        coreStartPromise: core.start(),
         fleetPluginStart: await resources.plugins.fleet.start(),
```

Comment:
should we also pass the fleet plugin start promise since we're passing a promise for core start? or should we `await` for both promises here? I feel like it should be consistent.

Reply:
Good point. I'll take a look